e2e/tests/assets/pod-with-pvc-thin-consumer.yaml

```diff
@@ -0,0 +1,46 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  namespace: default
+  name: test-claim-thin
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: topolvm-provisioner-thin
+  dataSource:
+    name: my-snap
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
```

Comment:
is this the only difference between two files? maybe we could have common `Pod` and separate `PVC`

Reply:
+1 was thinking that as well - was just expedient at the moment.  I'll fix this up.

Reply:
See #1966 and let me know what you think